css/components/hero.css

```diff
@@ -1,165 +1,187 @@
-.background-container{
-    background-color: var(--color-light-blue);
-    background-image: url('../../img/hero-bg.jpg');
-    background-repeat: no-repeat;
-    background-size:cover;
-    background-position: bottom left;
-    width: 100%;
-    height: auto;    
+.background-container {
+  background-color: var(--color-light-blue);
+  background-image: url("../../img/hero-bg.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: bottom left;
+  width: 100%;
+  height: auto;
+  position: relative;
+  padding-bottom: 100px;
+}
+.background-container::after {
+  content: "";
+  position: absolute;
+  bottom: -10%;
+  z-index: 1;
+  left: 0;
+  right: 0;
+  background-color: var(--color-white);
+  height: 30%;
+  clip-path: polygon(0 60%, 100% 0%, 100% 75%, 0% 100%);
```

Comment:
Čia visiškai smulkmena, bet 0 reikšmė vienur su % ženklu, kitur be, gal reikėtų suvienodinti (?)

Reply:
geras pastebėjimas :)